README.md

```diff
@@ -1 +1,148 @@
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# budget App <a name="about-project"></a>
+
+> Budget app is an app for managing transactions with categorized lists, sign-up and login pages, and navigation buttons for moving between pages. Users can add new categories or transactions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- Key Features -->
+
+## Key Features <a name="key-features"></a>
+
+- [x] Users can create an account and log in.
+- [x] Login users can create a new budget.
+- [x] Login users can add ingredients to a budget.
+- [x] Login users can delete a budget.
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `Ruby --version 3.2.1`
+- `Ruby on Rails --version ~> 7.0.4`
+- `Postgresql`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your_desired_folder
+  git clone https://github.com/ohyajk/budget-app.git
+```
+
+### Install
+
+Install this project with:
+
+- After cloning type `cd budget-app`.
+- Run `bundle install` to install all gems.
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `rails db:create` to create a new database
+- Run `rails db:migrate` to migrate the db and build the schema
+- Run `rails s` to start the server
+
+### Run tests
+
+- Run `rspec` to run the tests
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Jitender Kumar**
+
+- GitHub: [@ohyajk](https://github.com/ohyajk)
+- Twitter: [@ohyajk](https://twitter.com/@ohyajk)
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Create Purchases list**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ohyajk/budget-App/issues)
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+## Acknowledgments
+
```

Comment:
- [ ] Kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) in the "Acknowledgements" section or similar in your README as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.).

Reply:
Thank you for letting me know. I have added credit to [Gregoire Vella](https://www.behance.net/gregoirevella)